webapp/package.json

```diff
@@ -5,18 +5,28 @@
   "engines": {
     "node": ">= 6.0.0"
   },
-  "scripts": {},
+  "scripts": {
+    "build": "grunt",
+    "server": "grunt server",
+    "start": "grunt dev",
+    "test": "mocha --compilers coffee:coffee-script/register"
+  },
   "devDependencies": {
     "amdefine": ">=1.0.0",
+    "coffee-script": "^1.12.7",
+    "extend": "^3.0.1",
     "grunt": "~0.4.0",
     "grunt-banner": "^0.6.0",
+    "grunt-cli": "^1.2.0",
     "grunt-connect-proxy": "~0.2.0",
     "grunt-contrib-coffee": "~0.7.0",
     "grunt-contrib-connect": "~0.8.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-requirejs": "~1.0.0",
     "grunt-contrib-uglify": "~0.2.4",
-    "grunt-contrib-watch": "^0.6.1"
+    "grunt-contrib-watch": "^0.6.1",
+    "jquery": "^3.2.1",
```

Comment:
Do we really need jquery here? (just a question)

Reply:
Yes, because of following error:
![image](https://user-images.githubusercontent.com/9026019/32762605-b34ad4ce-c93d-11e7-82f7-2f9f65795419.png)
It appears without jquery.